src/string_util.cc

```diff
@@ -141,7 +143,7 @@ std::string StringPrintFImp(const char *msg, va_list args)
   // add 1 to size to account for null-byte in size cast to prevent overflow
   size = static_cast<std::size_t>(ret) + 1;
   auto buff_ptr = std::unique_ptr<char[]>(new char[size]);
-  ret = std::vsnprintf(buff_ptr.get(), size, msg, args);
+  ret = vsnprintf(buff_ptr.get(), size, msg, args);
```

Comment:
Leave a comment here with the date and why it can't be qualified with "std::". This way we prevent people from breaking it.